mathjs/mathjs.d.ts

```diff
@@ -1334,6 +1334,10 @@ declare namespace mathjs {
 	export interface MathNode {
                 isNode: boolean;
                 isSymbolNode: boolean;
+                isConstantNode: boolean;
+                isOperatorNode: boolean;
+                op: string;
```

Comment:
these are only visible on ConstantNode, and not all nodes. so either create a subclass, or make them optional.